src/main/resources/reference.conf

```diff
@@ -76,6 +79,18 @@ cassandra-journal {
   # The time to wait before cassandra will remove the thombstones created for deleted entries.
   # cfr. gc_grace_seconds table property documentation on http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
   gc-grace-seconds = 864000
+  
+  # Configure know tags in this section
```

Comment:
I have a solution in mind using clustering and duplication. I will try that and potentially introduce in a separate PR. It would complicate things such as reads and replay. I think having 3 tags is a reasonable tradeoff and we can update later if the solution is good.